test/config.js

```diff
@@ -12,6 +14,7 @@ describe('config.json schema', () => {
 
   before(() => {
     const ajv = new Ajv({allErrors: true});
+    ajv.addMetaSchema(ajvSchemaDraft06);
```

Comment:
Whats the reason this is needed?

Reply:
AJV 6 was updated to use draft-07 by default. Since AJV's schema's are draft-06 this is necessary.

The schema should probably be updated to draft-07 at some point but IMHO that can/should be a second step.

Reply:
ETA for this getting released?

Reply:
@gajus Would love to have this released as a patch as well-- I'm happy to help on anything else needed to make that happen ASAP.  This is currently causing NPM warnings for anyone with the latest version of eslint installed in their projects (or any 4.x release of eslint for that matter) -- and thus for every Sails v1 user with a newly generated project.  (Because `table` is a few levels deep, we can't pin it at the application level, and we don't want to confuse new Node users by including both a shrinkwrap and a package-lock file)

Reply:
Published.

Reply:
@gajus thank you! 

Unfortunately, eslint pinned itself to 4.0.2 (presumably due to some unusual usage of require?)  So for anyone else following this-- as of now, it is still a problem.  Hoping we can get it resolved: https://github.com/eslint/eslint/commit/6b71fd0bcbf9cc00ed4076587f5692b72f6e9aa5#commitcomment-28261398

In the mean time, I'll just fork and republish like I did with @sailshq/htmlhint and @sailshq/lodash so we make all the exciting warnings go away (hopefully for good).